docs/guide/custom-ui.md

```diff
@@ -0,0 +1,62 @@
+# Customizing the Apostrophe user interface
```

Comment:
I will probably rename this file to narrow its scope to "overriding admin UI Vue components" only. Larger topic is too big for this document although I need to do an overview guide as well which talks about apps vs components and the bus.